### PR TITLE
Integrating MAP routing (amp and canonical) with Cypress tests

### LINKED ISSUE #3455: Routing for Media Asset Pages in Simorgh
**Is your feature request related to a problem? Please describe.**
We need media asset pages on Simorgh for our first media asset pages
We need to build out pages to host media assets from CPS.  
We want to build pages where the page type is "MAP" only

- bbc.com/pidgin
- bbc.com/Igbo
- bbc.com/Yoruba
- bbc.com/Thai
- bbc.com/Punjabi 

Standard page: 
bbc.com/$CPSassetUri 
Amp Page: 
bbc.com/$CPSassetUri.amp

Example: 
MAP Page URL will typically look like this: https://www.test.bbc.com/pidgin/tori-49450859
OR this https://www.bbc.com/pidgin/media-49457815

**Describe the solution you'd like**
We want to visit a media asset page, have the request appropriately routed and a basic page rendered. 

The page should render:

- a header with the brand
- navigation 
- Footer 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

Cypress testing on the page on the existing components 
- [x] This feature is expected to need manual testing.

Given I visit https://www.test.bbc.com/pidgin/tori-49450859
Then I will see a page rendered with the brand header for pidgin
and the navigation for pidgin
and the footer for pidgin

**Additional context**

### PR DESCRIPTION
**Pull request by ibMadbouly**
Resolves #3455 
Continuing on PR https://github.com/bbc/simorgh/pull/3676 this PR is for building the cypress tests setup for MAP pages  

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- Added route configuration for the new page type MAP at `src/app/routes/index.js`
```js
  {
    path: mediaAssetPageRegexPath,
    exact: true,
    component: MediaPage,
    getInitialData: getMediaAssetPageInitialData,
    pageType: 'MAP',
  }
```

- at `cypress/Integration/pages/`: Added new  `pageType` folder for `mediaAssetPage` including these files `index.js` `tests` `testsForAMPOnly` `testsForCanonicalOnly` 

- at `cypress/support/config/services.js`: added the entry for `mediaAssetPage` for the service need to be e2e tested like `pidgin` like 
```js
mediaAssetPage: {
        path:
          Cypress.env('APP_ENV') === 'local'
            ? '/pidgin/tori-49450859'
            : undefined,
        smoke: true,
      }
```

- note: to put the same entry for all other services but no `path` and `smoke` to be `false`

```js
mediaAssetPage: { path: undefined, smoke: false }
```

- Commands : in a separate terminal window run `npm run build && npm start` and in another window to run specific e2e for specific `pageType` use the following command
`/node_modules/.bin/cypress run --spec cypress/integration/pages/mediaAssetPa
ge/index.js`
-  fixtures data added in the PR #3676 
- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

**Testing notes :**
 can try the following examples

- `http://localhost:7080/punjabi/international-49567825.amp`
-  `http://localhost:7080/punjabi/international-49567825.json`
-  `http://localhost:7080/punjabi/international-49567825`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/regex/index.test.js [hunk 1]
```diff
@@ -175,4 +177,49 @@ describe('mediaRadioAndTvRegexPathsArray', () => {
...
+  describe('mediaAssetPageDataRegexPath', () => {
+    const validRoutes = [
+      '/pidgin/test-49450859.json',
+      '/kyrgyz/test-tv-48695523.json',
+      '/mundo/test_underscore-12345678.json',
+    ];
+
+    shouldMatchValidRoutes(validRoutes, mediaAssetPageDataRegexPath);
+
+    // Accroding to CPS a valid assetUri should have 8 digits or more
```
Comment (jroebu14) [inline src/app/routes/regex/index.test.js]: typo here

**Review by jroebu14 — APPROVED** [review]
LGTM

**Review by ryanmccombe — APPROVED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`Tests for ${service} ${pageType}`, () => {});
```
Comment (pjlee11) [inline cypress/integration/pages/mediaAssetPage/tests.js]:
```suggestion
  describe(`No testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {});
```

File: cypress/integration/pages/mediaAssetPage/testsForAMPOnly.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForAMPOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForAMPOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForAMPOnly = ({
+  service,
+  pageType,
+}) => describe(`Amp Tests for ${service} ${pageType}`, () => {});
```
Comment (pjlee11) [inline cypress/integration/pages/mediaAssetPage/testsForAMPOnly.js]:
```suggestion
}) => describe(`No testsThatFollowSmokeTestConfigForAMPOnly to run for ${service} ${pageType}`, () => {});
```

File: cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
+  service,
+  pageType,
+}) =>
+  describe(`Canonical Tests for ${service} ${pageType}`, () => {
+    // TODO Chartbeat not yet implemented
+    describe('Chartbeat', () => {
```
Comment (pjlee11) [inline cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js]: I don't think this chartbeat test is valid for the live radio page.

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Just a couple of changes to cypress tests

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Pull request merged**